astro/src/components/download/DownloadWidget.astro

```diff
@@ -0,0 +1,624 @@
+    <ul class="flex mt-3 mx-4">
+      <li class="group active"><a data-tab-button href="#sandbox-link" class="block border-b-2 border-transparent group-[.active]:border-indigo-400 px-7 py-3 font-bold text-md text-slate-400 group-[.active]:text-white">Sandbox</a></li>
+    </ul>
+    <hr class="border-slate-700"/>
+    <div data-tab-content id="sandbox-link" class="m-8">
+      <h2 class="mb-4 text-2xl">Sandbox (running in FusionAuth Cloud)</h2>
+      <p class="mb-6">
+        Want to check out FusionAuth before installing? Check out our <a href="https://sandbox.fusionauth.io/" class={anchorClass}>sandbox environment</a>.
+      </p>
+      <p class="italic mb-6 text-xs">
+        P.S. our sandbox environment is running on <a href="https://fusionauth.io/platform/fusionauth-cloud" class={anchorClass} target="_blank">FusionAuth Cloud</a>.
+      </p>
+    </div>
+  </div>
+  <div data-tab-content id="marketplaces" class:list={[tabClass, "hidden"]}>
+    <ul class="flex mt-3 mx-4">
+      <li class="group active"><a data-tab-button href="#marketplaces-links" class="block border-b-2 border-transparent group-[.active]:border-indigo-400 px-7 py-3 font-bold text-md text-slate-400 group-[.active]:text-white">Marketplaces</a></li>
+    </ul>
+    <hr class="border-slate-700"/>
+    <div data-tab-content id="marketplaces-links" class="m-8">
+      <h2 class="mb-4 text-2xl">Marketplaces</h2>
+      <p class="mb-6">
+        FusionAuth can run on any server, anywhere, but is also available on public cloud provider’s marketplaces.
+      </p>
+      <p class="mb-6">
+        Choose a public cloud below for instructions on how to install using their marketplace.
+      </p>
+      <div class="flex justify-around">
+        <div class="text-center">
+          <a href="https://marketplace.digitalocean.com/apps/fusionauth" class="bg-white block border-2 border-transparent mb-4 p-4 rounded-lg w-36 hover:border-indigo-400">
+            <img src="/img/icons/download/digital-ocean-vertical.svg" alt="Digital Ocean"/>
+          </a>
+          <p>Digital Ocean</p>
+        </div>
+        <div class="text-center">
+          <a href="https://aws.amazon.com/marketplace/seller-profile?id=30ecc80d-f96e-4259-ae1c-fc3b994f90ea" class="bg-white block border-2 border-transparent mb-4 p-4 rounded-lg w-36 hover:border-indigo-400">
+            <img src="/img/icons/download/amazon-web-services.svg" alt="Amazon Web Services"/>
+          </a>
+          <p>Amazon Web Services</p>
+        </div>
+        <div class="text-center">
+          <a href="#" class="bg-slate-400 block border-2 border-transparent cursor-none mb-4 p-4 pointer-events-none rounded-lg w-36 hover:border-indigo-400">
+            <img src="/img/icons/download/google-cloud-platform.svg" alt="Google Cloud"/>
+          </a>
+          <p class="italic">Google (coming soon)</p>
```

Comment:
Here's the link for google cloud: https://console.cloud.google.com/marketplace/product/fusionauth-public/fusionauth-enterprise

From https://fusionauth.io/docs/get-started/run-in-the-cloud/marketplaces/google-cloud